view/frontend/web/js/view/payment/method-renderer/adyen-hpp-method.js

```diff
@@ -373,18 +390,31 @@ define(
             placeRedirectOrder: function(data) {
                 // Place Order but use our own redirect url after
                 var self = this;
+                fullScreenLoader.startLoader();
 
                 var messageContainer = this.messageContainer;
                 if(brandCode()) {
                     messageContainer = self.messageComponents['messages-' + brandCode()];
                 }
-                
+
+                $('.hpp-message').slideUp();
+
                 this.isPlaceOrderActionAllowed(false);
-                fullScreenLoader.startLoader();
                 $.when(
                     placeOrderAction(data, messageContainer)
                 ).fail(
-                    function () {
+                    function (response) {
+                        fullScreenLoader.stopLoader();
+                        //messageContainer.addErrorMessage((response['responseJSON'].message).replace('%1', response['responseJSON'].parameters[0]));
```

Comment:
Yes will do.
I tried to show the HPP messages in the Magento way but on HPP since we have more "regions" than payment methods in the xml defined it looks like it is not working. That's why at the end I just end up with JQuery.